python/pyarrow/io.pxd

```diff
@@ -25,6 +25,8 @@ from pyarrow.includes.libarrow_io cimport (ReadableFileInterface,
 cdef class Buffer:
     cdef:
         shared_ptr[CBuffer] buffer
+        Py_ssize_t shape[1]
+        Py_ssize_t strides[1]
```

Comment:
thanks -- on closer look the `Py_buffer.shape` and strides fields are pointers, so all good